test/bdd/fixtures/demo/docker-compose-auth.yml

```diff
@@ -0,0 +1,123 @@
+#
+# SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: '3'
+
+services:
+
+  auth.rest.example.com:
```

Comment:
In a followup, we should clean up the container names - to be more consistent about dashes and domains.